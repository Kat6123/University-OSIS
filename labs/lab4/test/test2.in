abracadabra
12234ffffff
asdsfsd
sdffff
6677556hhdhu6
