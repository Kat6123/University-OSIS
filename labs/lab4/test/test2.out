arbadacarba
ffffff43221
dsfsdsa
ffffds
6uhdhh6557766
